vagrant/Makefile

```diff
@@ -14,6 +14,9 @@ local_values_file = ${root_dir}vagrant/values.local.yaml
 name = collection
 namespace = sumologic
 
+test:
+	${root_dir}ci/run-tests.sh
```

Comment:
I'm not entirely sold on this name, I think this should be just `test.sh`

Reply:
... or `tests.sh`, just not `run-*`.

Reply:
I'll go with `tests.sh`